include/openssl/cms.h

```diff
@@ -85,7 +85,7 @@ ASN1_OCTET_STRING **CMS_get0_content(CMS_ContentInfo *cms);
 int CMS_is_detached(CMS_ContentInfo *cms);
 int CMS_set_detached(CMS_ContentInfo *cms, int detached);
 
-# ifdef HEADER_PEM_H
+# ifdef OPENSSL_PEM_H
```

Comment:
so, this means if you include pem.h after after cms.h then some symbols are missing.
cant we just include pem.h in the include section above?

Reply:
so this sequence fails to declare the function PEM_read_CMS and similar:
```
#include <openssl/cms.h>
#include <openssl/pem.h>
```
Assuming there is a reason why including pem.h from cms.h is intentionally not done,
one could fix that by adding a ifndef OPENSSL_CMS_H section in pem.h
which invokes the same DECLARE_PEM_rw_const macro.
likely as a followup PR.

Reply:
Still to be done...